src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -101,16 +105,35 @@ def create_or_update(self, run: Run, **kwargs) -> Run:
         except RunExistsError:
             raise RunExistsError(f"Run {run.name!r} already exists.")
 
+    def _create_run_from_existing_run_folder(self, run: Run, **kwargs) -> Run:
+        """Create run from existing run folder."""
+        try:
+            self.get(run.name)
+        except RunNotFoundError:
+            pass
+        else:
+            raise RunExistsError(f"Run {run.name!r} already exists.")
+
+        try:
+            run._dump()
+            return run
+        except Exception as e:
+            raise UserErrorException(
+                f"Failed to create run {run.name!r} from existing run folder {run.source!r}: {str(e)}"
+            ) from e
+
     def _print_run_summary(self, run: Run) -> None:
-        print("======= Run Summary =======\n")
-        duration = str(run._end_time - run._created_on)
-        print(
-            f'Run name: "{run.name}"\n'
-            f'Run status: "{run.status}"\n'
-            f'Start time: "{run._created_on}"\n'
-            f'Duration: "{duration}"\n'
-            f'Output path: "{run._output_path}"\n'
-        )
+        # print run summary if they run is not from existing run folder
```

Comment:
why we skip print summary for existing run? is there any missing information?

Reply:
Good catch, currently the downloaded run info has no run meta info like start time and endtime, so the time calculation here is not invalid. 

The flow artifacts have child run's start time and end time, I'm not sure if it's OK to pick the start time of first run and end time of the last run and regard it as the duration of the run record.

Reply:
Recovered, now we have start time and end time.